site/content/en/latest/tasks/security/restrict-ip-access.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "IP Whitelisting/Blacklisting"
```

Comment:
lets replace these terms as highlighted by the CNCF INI doc
https://inclusivenaming.org/word-lists/tier-1/whitelist/